### DISCUSSION TITLE
Lack of meta text field to boards in Beta version

### DISCUSSION THREAD

**Discussion by ah-tjek** [community/community#9432] [Other Feature Feedback, Questions, & Ideas]
I need a text field for my beta version board - like the one we have at the front in the "old version". 

I have used it to anchor the sprint goal, sprint agenda, links to documents we use in every sprint and what not.

**Comment by brianlovin** [comment]
Hey @ah-tjek I think this might be a request related to Issues — could you post over here so that the correct team is able to reply? Thanks!

https://github.com/github/feedback/discussions/categories/issues-feedback
